### ISSUE TITLE
elem attributes not reacting to javelin cell for :bc or :b tag

### ISSUE THREAD

**Issue by charkcl** [hoplon/ui#9]
To keep track of the issue discussed on slack already:)

Using javelin formula cell as an attribute value returns below error in browser(Chrome) console
`No protocol method IAttr.-dom-attribute defined for type javelin.core/Cell: [object Object]`

Snippet of example code below:

``` cljs
(def curr (cell true))
(defelem test-section
  "Consists of a button that should change border color of the box below the button"
  [attrs elems]
  (elem :sh (r 1 1)
    (elem :sh (r 1 1) :m :pointer "button to change color" :click #(reset! curr false))
    (elem :sh (r 1 1) :bc (cell= (if curr (c 0x000000) (c 0x2196F3))) :b 1 "border color of the box")))
```

**Comment by adamwielowieyski** [comment]
Repro repo: https://github.com/addplus/tpl-example

**Comment by jumblerg** [comment]
this should have been resolved by #de96df2.
